Birdui/Birdui/PostListView.swift

```diff
@@ -9,10 +9,56 @@
 import SwiftUI
 
 struct PostListView: View {
+    @State var modalIsPresented = false
+
+    struct HeaderTitleTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .foregroundColor(Color.black)
+                .font(.system(size: 17))
+        }
+    }
+
+    struct ButtonTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .font(.system(size: 15))
+        }
+    }
+
     var body: some View {
-        // TODO: This should look exactly like the Birdie table view,
-        // but with only one button.
-        Text("Layout header, new-post button, List of posts")
+        NavigationView {
```

Comment:
Why do you need to use NavigationView? The bar title is hidden on lines 52-53 anyways.

Reply:
I was thinking that we need navigationView to open the modal view